sql/parser/sql.y

```diff
@@ -830,10 +835,26 @@ var_value:
   }
 
 iso_level:
-  READ UNCOMMITTED {}
-| READ COMMITTED {}
-| REPEATABLE READ {}
-| SERIALIZABLE {}
+  READ UNCOMMITTED
```

Comment:
I was surprised to see these still in our grammar; it might be worth a comment to say that we support these for compatibility and map them to our closest equivalent.